portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/security/openidconnect/dependencies/com.liferay.portal.security.sso.openid.connect.internal.configuration.OpenIdConnectProviderConfiguration~KeycloakPKCE.config

```diff
@@ -4,8 +4,8 @@ discoveryEndPointCacheInMillis=L"360000"
 idTokenSigningAlgValues=["RS256"]
 issuerURL="http://127.0.0.1:18080/auth/realms/master"
 jwksURI="http://127.0.0.1:18080/auth/realms/master/protocol/openid-connect/certs"
-openIdConnectClientId="liferay-pkce"
-openIdConnectClientSecret="bd6782f9-69d9-45a3-ad33-b2daaedebf08"
+openIdConnectClientId="${openid.provider.keycloak.client.id.2}"
+openIdConnectClientSecret="${openid.provider.keycloak.client.secret.2}"
```

Comment:
Should be openid.provider.keycloak.password.2: https://github.com/liferay/liferay-jenkins-ee/blob/master/commands/build.properties#L1108